admin-dev/themes/new-theme/scss/components/_category_tree.scss

```diff
@@ -91,6 +91,13 @@ ul.category-tree {
     }
   }
 
+  label.form-control-label {
+    color: $gray-medium;
+    width: 100%;
+    padding-left: 15px;
```

Comment:
Please prefer use rem rather than px

Reply:
Yes, I thought about doing that but as the list just below this label is placed in a `.col-sm` div which is itself padded by 15px I thought I should keep it that way.
Wdyt?